tox.ini

```diff
@@ -40,11 +40,11 @@ commands = black .
 [testenv:benchmarks]
 description = Dry run the benchmarks to check for errors
 basepython = python3
-deps = asv
+deps = git+https://github.com/airspeed-velocity/asv.git@master#egg=asv
```

Comment:
I could imagine that depening on master could lead to some errors at some later point. One alternative would be to specify the exact commit that we want to depend on: 
```
deps = git+https://github.com/airspeed-velocity/asv.git@a131ca742daf53c4521fbfefeb53507b5c140c84#egg=asv
```
Tested locally this seems to work. On the other hand, I also don't think it's reall big deal to depend on master, even if at some point this CI check might fail, we handled similar problems fine :)

Reply:
Good call. Once the new release is out on PyPi we can pin this to a version.